CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v1.0.0-beta2 (unreleased)
+
+* [Breaking] Rename `modules/rpc` to `modules/yarpc`
```

Comment:
I think we should leave a note about the yaml change that needs to happen. I think that's the way YARPC has been writing changelog notes - with an action that needs to be taken, or more detailed information of what the rename actually entails.